src/contrib/cluster/Akka.Cluster.Sharding/Delivery/ShardingProducerController.cs

```diff
@@ -119,14 +122,22 @@ public void AskNextTo(MessageWithConfirmation<T> msgWithConfirmation)
 
     public sealed record Settings
     {
-        public Settings(int bufferSize, TimeSpan internalAskTimeout, TimeSpan cleanupUnusedAfter,
+        private Settings(int bufferSize, TimeSpan internalAskTimeout, TimeSpan cleanupUnusedAfter,
             TimeSpan resendFirstUnconfirmedIdleTimeout, ProducerController.Settings producerControllerSettings)
         {
             BufferSize = bufferSize;
             InternalAskTimeout = internalAskTimeout;
             CleanupUnusedAfter = cleanupUnusedAfter;
             ResendFirstUnconfirmedIdleTimeout = resendFirstUnconfirmedIdleTimeout;
             ProducerControllerSettings = producerControllerSettings;
+
+            // TODO: enable chunking in Akka.Cluster.Sharding.ProducerController
+            if (ProducerControllerSettings.ChunkLargeMessagesBytes is { } or > 0)
```

Comment:
Investigated the source of the https://github.com/akkadotnet/akka.net/pull/6720#issuecomment-1536758091 - as it turns out, sharding doesn't support chunking the JVM design. I suspect this is because the `State<T>` type for the `ShardingProducerController` isn't built for it.